discovery/bootstrapper.go

```diff
@@ -379,135 +379,135 @@ func (d *DNSSeedBootstrapper) SampleNodeAddrs(numAddrs uint32,
 
 	var netAddrs []*lnwire.NetAddress
 
-	// We'll continue this loop until we reach our target address limit.
-	// Each SRV query to the seed will return 25 random nodes, so we can
-	// continue to query until we reach our target.
+	// We'll try all the registered DNS seeds, exiting early if one of them
+	// gives us all the peers we need.
+	//
+	// TODO(roasbeef): should combine results from both
 search:
-	for uint32(len(netAddrs)) < numAddrs {
-		for _, dnsSeedTuple := range d.dnsSeeds {
-			// We'll first query the seed with an SRV record so we
-			// can obtain a random sample of the encoded public
-			// keys of nodes. We use the lndLookupSRV function for
-			// this task.
-			primarySeed := dnsSeedTuple[0]
-			_, addrs, err := d.net.LookupSRV("nodes", "tcp", primarySeed)
-			if err != nil {
-				log.Tracef("Unable to lookup SRV records via "+
-					"primary seed: %v", err)
+	for _, dnsSeedTuple := range d.dnsSeeds {
+		// We'll first query the seed with an SRV record so we can
+		// obtain a random sample of the encoded public keys of nodes.
+		// We use the lndLookupSRV function for this task.
+		primarySeed := dnsSeedTuple[0]
+		_, addrs, err := d.net.LookupSRV("nodes", "tcp", primarySeed)
+		if err != nil {
+			log.Tracef("Unable to lookup SRV records via "+
+				"primary seed (%v): %v", primarySeed, err)
 
-				log.Trace("Falling back to secondary")
+			log.Trace("Falling back to secondary")
 
-				// If the host of the secondary seed is blank,
-				// then we'll bail here as we can't proceed.
-				if dnsSeedTuple[1] == "" {
-					return nil, fmt.Errorf("Secondary seed is blank")
-				}
+			// If the host of the secondary seed is blank, then
+			// we'll bail here as we can't proceed.
+			if dnsSeedTuple[1] == "" {
+				log.Errorf("dns seed %v has no secondary, "+
```

Comment:
Nit: s/dns/DNS/

Reply:
Also don't see why this needs to be logged as an error? Maybe just keep trace as the one above?